### PR TITLE
[MeshModel][Docs] Add supported relationships screenshots

### PR DESCRIPTION
**Pull request by Boombag0607**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/assets/images/network_edge_relationship_ingress_service.png
Comment (leecalcote) [inline .github/assets/images/network_edge_relationship_ingress_service.png:1]: When @Abhishek-kumar09 addresses this bug (wrong namespace coloring), we'll want to take a new screenshot for this relationship type.

**Review by leecalcote — CHANGES_REQUESTED** [review]
@Boombag0607 thanks! Unless these are going into the readme, please put these screenshots into the /docs/<somewhere>/... folder.

**Comment by Boombag0607** [comment]
Added a quick fix to include all screenshots in contribution docs.

**Comment by l5io** [comment]
🚀 Preview for commit f7803bfa3aae07ca642e8b68ff959aba27c221c6 at: https://64c1fee65f34b44bb7dfeeef--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit cc310141fbc9240a49b916fef0b272db1191abf5 at: https://64c20d39e696c653ce67dc75--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 2e26245d5afb145367ac6571a4a00fdf62c6879b at: https://64c221e3c83715583d043c28--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]

**Pull request merged**